src/events/rawGuildMemberUpdate.ts

```diff
@@ -22,34 +23,65 @@ export default class extends Event {
 		const member = guild.members.get(data.user.id);
 		if (typeof member !== 'undefined') member._patch(data);
 
-		this.handleNicknameChange(guild, data);
+		this.handleMemberChange(guild, data);
 		floatPromise(this, this.handleRoleSets(guild, data));
 	}
 
-	private handleNicknameChange(guild: KlasaGuild, data: WSGuildMemberUpdate) {
-		// Get the current nickname, compare them both, if they are different, it changed
+	private handleMemberChange(guild: KlasaGuild, data: WSGuildMemberUpdate) {
+
+		// Look up which logs should be send in the guild database
+		let shouldLogNickname = guild.settings.get(GuildSettings.Events.MemberNicknameUpdate);
+		let shouldSendRoleLog = guild.settings.get(GuildSettings.Events.MemberRoleUpdate);
```

Comment:
Those should be moved as they're not used until a few lines below, and can end up unused as of line 50:

```javascript
if (typeof previous === 'undefined') return;
```